frontend/src/api/Room.ts

```diff
@@ -13,7 +13,6 @@ export type CreateRoomParams = {
 };
 
 export type JoinRoomParams = {
-  roomId: string,
   user: User,
```

Comment:
This isn't much of an issue but is it weird that there is only one field in the params? This also happens in several other places since we took out roomId as a path variable.

Reply:
Yeah I think this is fine, I don't think the number of fields in the params really matters too much, the object does what its supposed to.